plasmapy/formulary/frequencies.py

```diff
@@ -516,3 +519,112 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m**-3) -> u.rad / u.s:
+    ion2 : `~plasmapy.particles.particle_class.Particle`
+        Representation of ion species #2 (same behavior as for ion1).
+    Z1 : `float` or `~astropy.units.Quantity`, optional
+        The charge state for ion species #1. If not provided, it defaults to the charge number of ``ion1``.
+    Z2 : `float` or `~astropy.units.Quantity`, optional
+        The charge state for ion species #2. If not provided, it defaults to the charge number of ``ion2``.
+
+    Returns
+    -------
+    omega_BB : `~astropy.units.Quantity`
+        The Buchsbaum frequency of the plasma in units of radians per second.
+        Setting keyword ``to_hz=True`` will apply the factor of :math:`1/2π`
+        and yield a value in Hz.
+
+    Raises
+    ------
+    `TypeError`
+        If the magnetic field is not a `~astropy.units.Quantity` or
+        ``particle`` is not of an appropriate type.
+    `ValueError`
+        If the magnetic field contains invalid values or particle cannot
+        be used to identify a particle or isotope.
+
+    Warns
+    -----
+    : `~astropy.units.UnitsWarning`
+        If units are not provided, SI units are assumed.
+
+    Notes
+    -----
+    In a magnetized plasma, the presence of two ion species allows the
+    perpendicular component of the cold-plasma dielectric coefficient
+    :math:`\epsilon_{\perp}` to vanish at an angular frequency referred
+    to as the Buchsbaum frequency :cite:p:`buchsbaum:1960`, also called
+    the bi-ion hybrid resonance frequency :cite:p: `thompson:1995`, or ion-ion hybrid frequency
+    :cite:p:`vincena:2013`.  This frequency can be defined as:
+
+    .. math::
+        \omega_{BB} \equiv \sqrt{\frac{\omega_{p1}^{2}\omega_{c2}^{2}
+            + \omega_{p2}^{2}\omega_{c1}^{2}}{\omega_{p2}^{2}+\omega_{p2}^{2}}}
+
+    Examples
+    --------
+    >>> from astropy import units as u
+    >>> fbb = Buchsbaum_frequency(0.1*u.T, 1e18*u.m**-3, 1e18*u.m**-3, "proton", "He+", to_hz=True)
+    >>> fbb
+    <Quantity 764831.28372462 Hz>
+    >>> fc_helium = gyrofrequency(0.1*u.T, "He+", to_hz=True)
+    >>> fc_proton = gyrofrequency(0.1*u.T, "proton", to_hz=True)
+    >>> fbb/fc_helium
+    <Quantity 1.99327444>
+    >>> fbb/fc_proton
+    <Quantity 0.50168706>
+
+    """
+
+    omega_c1_squared = gyrofrequency(B, ion1, signed=False, Z=Z1)**2
+    omega_c2_squared = gyrofrequency(B, ion2, signed=False, Z=Z2)**2
+    omega_p1_squared = plasma_frequency(n1, ion1, z_mean=Z1)**2
+    omega_p2_squared = plasma_frequency(n2, ion2, z_mean=Z2)**2
+
+    return np.sqrt((omega_p1_squared * omega_c2_squared + omega_p2_squared * omega_c1_squared) /
+                   (omega_p1_squared + omega_p2_squared))
+
+
+wbb_ = Buchsbaum_frequency
+"""Alias to `~plasmapy.formulary.frequencies.Buchsbaum_frequency`."""
+
+wii_ = Buchsbaum_frequency
+"""Alias to `~plasmapy.formulary.frequencies.Buchsbaum_frequency`."""
+
+wbi_ = Buchsbaum_frequency
+"""Alias to `~plasmapy.formulary.frequencies.Buchsbaum_frequency`."""
```

Comment:
I've never been a fan of those aliases, but leaving that aside - do you think it's a good idea to have three aliases point to this same function? Smells like an antipattern.

Reply:
This is not a fundamental quantity. I removed the aliases. Thanks.